tfhe/src/shortint/parameters/mod.rs

```diff
@@ -131,11 +118,195 @@ impl Parameters {
     }
 }
 
+#[derive(Serialize, Copy, Clone, Deserialize, Debug, PartialEq)]
+pub enum ShortintParameterSetInner {
```

Comment:
I don't think this needs to be public, especially since the `inner` member in  `ShortintParameterSet` is not

I think either both are public or none are, unlesss i'm missing something ?

Reply:
yeah I realized that after pushing